Cargo.toml

```diff
@@ -12,20 +12,18 @@ license = "GPL-3.0"
 repository = "https://github.com/tweedegolf/lzjd-rs"
 
 [dependencies]
-clap = "2.32.0"
-base64 = "0.10.1"
-failure = "0.1.5"
-failure_derive = "0.1.5"
-fasthash= "0.4.0"
+clap = "3.2"
+base64 = "0.13.1"
+murmurhash3 = { git = "https://github.com/malwaredb/murmurhash3-rs", branch = "update" }
```

Comment:
This will prevent publishing to crates.io.

Reply:
I know, that's another abandoned crate.

I don't know what I was expecting with this PR. Maybe some acknowledgement from the maintainer. The abandoned crates thing bothers me.

Reply:
@hdoordt Maybe you can shed some light. Maybe you can transfer maintainership or get a grant for it?